test/libsolidity/syntaxTests/natspec/docstring_enum_value_definition.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+	enum Color {
+		Red,
+		/// @title example of title
+		/// @author example of author
```

Comment:
You can remove the title and author tags from here, pretty sure no one's gonna add them to enum value definitions.